caddyhttp/markdown/metadata/metadata.go

```diff
@@ -68,7 +68,7 @@ func (m *Metadata) load(parsedMap map[string]interface{}) {
 		switch v := val.(type) {
 		case bool:
 			m.Flags[key] = v
-		case string:
+		default:
```

Comment:
This is great. Do you know why we separate Flags and Variables? (/cc @abiosoft) - do they need to stay separate?